benchmarks/framework_comparison/random_walks/results.md

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Can you please add your setup in the file before your results (M chip, mlx/mlx-graphs/pyg/numpy versions)